src/main/java/com/squareup/javapoet/ClassName.java

```diff
@@ -138,6 +138,14 @@ public String simpleName() {
     return simpleName;
   }
 
+  /**
+   * Returns the full class name of this class, like {@code "java.util.Map.Entry"}
```

Comment:
Can we put a period between the description and the example? This will prevent the example from showing in the short Javadoc in the method table and the IDE unless you click/expand it for more.

Reply:
I tried to make this JavaDoc looks like JavaDoc for `simpleName()` where example goes right after the comma.

Reply:
We should fix that one too! I can do it separately.